snoop/data/analyzers/image_classification.py

```diff
@@ -0,0 +1,147 @@
+"""Task to call a service that runs object detection and/or image classification on images"""
+
+import requests
```

Comment:
generally you want to sort imports like this:
- python std libs first, like json and io
- external libs second, like requiests, PIL, django
- project relative imports last, like ..tasks,  ..models

it doesn't usually matter, but it helps to be organized